app/assets/stylesheets/confirm.scss

```diff
@@ -0,0 +1,5 @@
+.confirm {
+  .message {
+    margin-top: 40px;
+  }
+}
```

Comment:
Do we need a special stylesheet just for one rule? :)

Reply:
Moved into custom.scss